source: in-house
Set dbus-daemon path to (non-standard) /usr/lib.

--- a/installer/core_install.py
+++ b/installer/core_install.py
@@ -382,7 +382,7 @@ 'xsane':            (False, ['scan'], XS
             'scanimage':        (False, ['scan'], SCANIMAGE_STR, self.check_scanimage, DEPENDENCY_RUN_TIME, '1.0', 'scanimage --version', EXTERNALDEP),
 
             # Required fax packages
-            'dbus':             (True,  ['fax'], DBUS_STR, self.check_dbus, DEPENDENCY_RUN_AND_COMPILE_TIME, '-', 'dbus-daemon --version', EXTERNALDEP),
+            'dbus':             (True,  ['fax'], DBUS_STR, self.check_dbus, DEPENDENCY_RUN_AND_COMPILE_TIME, '-', '/usr/lib/dbus-daemon --version', EXTERNALDEP),
 
             # Required and optional qt4 GUI packages
             # optional for non-sudo behavior of plugins (only optional for Qt4
